### DISCUSSION TITLE
Deadliner, gently manage your deadline

### DISCUSSION THREAD

**Discussion by YassinEldeeb** [emilk/egui#1453] [Show and tell]
🎉🥳Announcing Deadliner

Manage your deadline gently by updating the wallpaper of your desktop with the time left.

🗓️Set deadline
🖼️Update wallpaper dynamically
🤗Less annoying way for reminding you
🎨Full Customization
👨‍👦‍👦Cross Platform
⚡Efficient

Landing page:
https://deadliner.vercel.app

Learn How it works:
https://youtu.be/ySn-Ykdq5BU

Source code:
https://github.com/YassinEldeeb/Deadliner

**Comment by emilk** [comment]
Oh wow, that looks cool! Please add a screenshot and al link in https://github.com/emilk/egui/issues/996 as well!

**Reply by YassinEldeeb** [reply]
Thank you, hope it gets some attention cause it has barely been seen by 4 people.
